tests/circle/anchors/test.typ

```diff
@@ -0,0 +1,21 @@
+#set page(width: auto, height: auto)
+#import "/src/lib.typ": *
+
+#box(stroke: 2pt + red, canvas({
+  import draw: *
+  
+  circle((0,0), radius: (2,3), name: "c")
+  for-each-anchor("c", name => {
+    content("c."+name, box(fill: white)[#name])
+  })
+}))
+
+#box(stroke: 2pt + red, canvas({
+  import draw: *
+  
+  rotate(45deg)
+  circle((0,0), radius: (2,3), name: "c")
+  for-each-anchor("c", name => {
+    content("c."+name, box(fill: white)[#name])
```

Comment:
I think the bounds are wrong for the bottom-left and top-right labels, they shouldn't be overlapping the edges...

Reply:
Oh, yes. There is a problem with bounds of `content` getting rotated when it should not.